manticore/native/cpu/abstractcpu.py

```diff
@@ -392,7 +392,7 @@ def invoke(self, model, prefix_args=None):
 platform_logger = logging.getLogger("manticore.platforms.platform")
 
 
-def unsigned_hexlify(i):
+def unsigned_hexlify(i: Any) -> Any:
```

Comment:
Yeah, why no `SupportsStr` type?  Probably because _everything_ in Python can be converted to a string — `__str__` is a method on `object`.

Additionally, the type of `unsigned_hexlify` is difficult to pin down precisely — if it is given a non-`int` argument, `unsigned_hexlify` is the identify function.  Otherwise, `unsigned_hexlify` returns a `str`.

I added the explicit signature again there, silly as it is, so that `mypy` would check the body of function, which it wouldn't otherwise do in the absence of type hints.